specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -1,4 +1,4 @@
-import "@typespec/versioning";
```

Comment:
Can we restore the above package.json and package-lock.json files please, those should not be touched :)

Reply:
Hi Libba,
Ran the command and committed the changes.

Reply:
Can you check them out from upstream/main we don't want any diff's in these files -- they are only updated by the tool owners.

Reply:
Hi Libba,
Can you please explain a bit. I followed the instructions from the https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/31175/Updating-Event-Grid-SDK-Swagger

and ran these commands to generate the json files 
![image](https://github.com/user-attachments/assets/bb668236-c210-46e7-8a32-9cd4792f9c3d)

then uploaded the changes.

Reply:
Yes these are the correct steps to follow. The package.json and package-lock.json are controlled by this repo's maintainers, we don't want to commit/merge any changes to these specific files when updating or generating eventgrid. I can add a note to the doc mentioning that. You should only commit the updated TypeSpec and Swagger files.

Reply:
Ohh okay okay.. I should not commit the package file.. got it.